md_test.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import pathlib
+import psutil
+import re
+import subprocess as sp
+import sys
+import time
+
+MD = "md0"
+MD_DEV = f"/dev/{MD}"
+verbose = False
+
+def debug(*args):
+    if verbose:
+        print(">", *args)
+
+def run(cmd, **kws):
+    debug(f"{' '.join(cmd)}")
+    return sp.check_output(cmd, **kws)
+
+# Find all unmounted non-Eideticom NVMe devices
+def find_devices():
+    devs = sp.check_output(["nvme", "list", "-o", "json"])
+    devs = json.loads(devs)
+    devs = filter(lambda d: "Eideticom" not in d["ModelNumber"], devs["Devices"])
+    devs = [d["DevicePath"] for d in devs]
+    debug(f"Non-Eideticom devices found: {' '.join(devs)}")
```

Comment:
This is a public repo.... Not sure I like this Eideticom specific noise creeping out... Is there another way we can handle this?

Reply:
Maybe add a `--skip-devices` option so it uses all devices it finds except those in the list?